cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -859,7 +934,7 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
                     tabs[1].editors = [StaffLockEditor];
                 } else if (xblockInfo.isSequential()) {
                     tabs[0].editors = [ReleaseDateEditor, GradingEditor, DueDateEditor];
-                    tabs[1].editors = [ContentVisibilityEditor];
+                    tabs[1].editors = [ContentVisibilityEditor, ShowCorrectnessEditor];
```

Comment:
@pomegranited I would suggest making it all 14px and increasing the padding between the radio button text and the helper text.